docs/users/step-debugging.md

```diff
@@ -8,7 +8,7 @@ All IDEs basically work the same: They listen on a port and react when they're c
 
 **Key facts:**
 
-* Enable xdebug with `ddev xdebug` or `ddev xdebug on`. It will remain enabled until you start or restart the project.
+* Enable xdebug by running `ddev xdebug` or `ddev xdebug on` in your docroot directory. It will remain enabled until you start or restart the project.
```

Comment:
```suggestion
* Enable xdebug by running `ddev xdebug` or `ddev xdebug on` in your project directory. It will remain enabled until you start or restart the project.
```